src/main/java/com/google/sps/data/AuthStatus.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+* Class object define the Authentication status of a user
+*/
+public class AuthStatus {
+  private String url;
+  private boolean isLogged;
+  private boolean isRegistered;
+  
+  /** Constructor() */
+  public AuthStatus() {
+    this.url = "";
+    this.isLogged = false;
+    this.isRegistered = false;
+  }
+  
+  /** Constructor(String, Boolean, Boolean) */
```

Comment:
Nit: I'm not very familiar with Java so I'm not sure, but I am not familiar with this type of documentation. I would have expected something like the following:

`/**
    * @param {string} url
    * @param {boolean} isLogged
    * @param {boolean} is isRegistered
    */`

Reply:
(Sorry about the formatting by the way, I guess GitHub doesn't like newlines)

Reply:
Actually being that data types are already included on the constructor I think I'll delete the comment but thanks for pointing it out Aki!